### DISCUSSION TITLE
Scrolling not working correctly

### DISCUSSION THREAD

**Discussion by ddkwork** [cogentcore/core#791] [General]
When the horizontal and vertical scroll bars appear in the layout at the same time, when the mouse wheel scrolls up and down, the horizontal scroll bar should be manually dragged instead of the horizontal scroll bar scrolling at the same time as the vertical scroll bar is scrolling, because when this behavior occurs, the controls in the entire layout look at the center of the window or the right side of the window, and cannot visually preview all the controls on the layout. Recurrence: This can be seen by scrolling the mouse after loading the code on the app's global settings page, code editor, or gide's split layout.

**Comment by kkoreilly** [comment]
I am aware of this issue and will fix it soon.

**Comment by kkoreilly** [comment]
Fixed in 739eb36370769e3abebea6a2806d220f9b67a7fe (see #797).

**Comment by ddkwork** [comment]
> Fixed in 739eb36370769e3abebea6a2806d220f9b67a7fe (see #797).

![Screenshot_20240115_004854_com.realvnc.viewer.android.jpg](https://github.com/goki/goki/assets/19886504/b272d1e5-db69-4a3f-a84f-0a1c4560ed00)

There is also a problem with the scrollbar: the horizontal scrollbar looks too high, and the drag event is triggered when the cursor points to the drawing area of the horizontal scrollbar, shouldn't it be that the cursor is valid as long as it is within the drawing area of the horizontal scrollbar? For the vertical scrollbar, the cursor cannot trigger the drag behavior of the left mouse button press in any area, only the mouse wheel.

**Reply by kkoreilly** [reply]
I will look into this.

**Reply by kkoreilly** [reply]
I filed #818 for this.
